### PR TITLE
Docs: add Translate FR guide for Generic

### PR DESCRIPTION
**Pull request by deoxis9001**
## What is this fixing or adding?

Docs: add Translate FR guide for Generic

### CODE REVIEW

File: worlds/generic/docs/commands_fr.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+- `!help` Renvoie une liste des commandes disponibles.
+- `!license` Renvoie les informations relatives à la licence du logiciel.
+- `!options` Renvoie les options actuelles du serveur, y compris le mot de passe en clair.
+- `!players` Renvoie des informations sur les joueurs connectés et non connectés.
+- `!status` Renvoie des informations sur l'état de la connexion et le nombre de vérifications effectuées pour tous les joueurs de la salle en cours. <br /> ( En option, mentionnez un nom de joueur et obtenez des informations sur ce joueurs. Par exemple : !status DeathLink)
+
+
+### Utilitaires
+- `!countdown <nombre de secondes>` Démarre un compte à rebours en utilisant la valeur de secondes donnée. Utile pour synchroniser les démarrages.
+  Defaults to 10 seconds if no argument is provided.
```
Comment (ReverM) [inline worlds/generic/docs/commands_fr.md:27]:
```suggestion
  Le compte à rebours durera 10 secondes si vous tapez simplement `!countdown` sans argument.
```

File: worlds/generic/docs/commands_fr.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- `/help` Renvoie une liste des commandes disponibles.
+- `/license` Renvoie les informations relatives à la licence du logiciel.
+- `/options` Renvoie les options actuelles du serveur, y compris le mot de passe en clair.
+- `/players` Renvoie des informations sur les joueurs connectés et non connectés.
+- `/save` Sauvegarde l'état du multi-monde actuel. Notez que le serveur procède à des sauvegardes automatiques toutes les minutes.
+- `/exit` Arrêt du serveur
+
+### Utilitaires
+- `/countdown <nombre de secondes>` Démarre un compte à rebours en utilisant la valeur de secondes donnée. Utile pour synchroniser les démarrages.
+  Defaults to 10 seconds if no argument is provided.
```
Comment (ReverM) [inline worlds/generic/docs/commands_fr.md:61]: Pareil que plus haut.

File: worlds/generic/docs/mac_fr.md
Comment (ReverM) [inline worlds/generic/docs/mac_fr.md:1]: Je ne suis pas certain à 100% que le guide de base macOS est à jour. Il faudrait alors le revérifier avant toute traduction.

**Review by ReverM — CHANGES_REQUESTED** [review]
Deux lignes n'ont pas été traduites dans le guide de commandes.

Le guide pour macOS n'a pas été révisé puisque je ne suis pas 100% certain si le guide de base est à jour.

**Comment by black-sliver** [comment]
I think this might need consideration in UI/UX so @LegendaryLinux

**Review by agilbert1412 — APPROVED** [review]
My comments are all very minor.
I looked at everything except Mac, Plando and triggers, which I don't know enough to validate

**Comment by deoxis9001** [comment]
> I think this might need consideration in UI/UX so @LegendaryLinux

what's the status of mac os documentation?

**Review by ThePhar — CHANGES_REQUESTED** [review]
I think until we have a better UX solution for displaying titles in different languages, these should be combined like all other setup docs handle it.

Example:
image

Instead of the following:
image

**Comment by deoxis9001** [comment]
> I think until we have a better UX solution for displaying titles in different languages, these should be combined like all other setup docs handle it.
> 
> Example: image
> 
> Instead of the following: image

On DLCQuest, I was told to follow the example of “Adventure”.

image

**Comment by deoxis9001** [comment]
I think I'll close this request if I don't get an answer.

**Comment by agilbert1412** [comment]
I'll update the tags for you, but for what it's worth, I've personally started closing my PRs if they get blocked over something minor and follow up questions are ignored.

**Comment by deoxis9001** [comment]
> I'll update the tags for you, but for what it's worth, I've personally started closing my PRs if they get blocked over something minor and follow up questions are ignored.

I'm really hesitating to close the PR and open it again on a blank PR.

**Review by agilbert1412 — APPROVED** [review]

**Pull request closed without merging**